pom.xml

```diff
@@ -32,12 +32,12 @@ under the License.
   <version>${revision}.${changelist}</version>
   <packaging>hpi</packaging>
   <name>SAML Plugin</name>
-  <url>https://github.com/jenkinsci/saml-plugin</url>
+  <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
```

Comment:
the artifact will never change, and the change is unrelated to the bump of the test harness dependency, however, the change is innocuous so we can keep it.

Reply:
> the artifact will never change, and the change is unrelated to the bump of the test harness dependency, however, the change is innocuous so we can keep it.

Sorry, I should have been more descriptive about this change. This was needed so I could test the incremental build with Plugin Compatibility Tester (PCT). Please see the following links:

- [JENKINS-58716](https://issues.jenkins.io/browse/JENKINS-58716)
- https://github.com/jenkinsci/archetypes/pull/192
- https://github.com/jenkinsci/incrementals-tools/pull/16
- https://github.com/jenkinsci/plugin-compat-tester/pull/181
- https://github.com/jenkins-infra/community-functions/pull/23